### DISCUSSION TITLE
案例《谁死于玛利亚飓风》

### DISCUSSION THREAD

**Discussion by wanfanghou** [data-journalism/data-journalism.github.io#31] [Show and tell]
![image](https://user-images.githubusercontent.com/543384/132837436-83639b08-1902-43ec-ba44-d652b6e6d5d7.png)

《谁死于玛利亚飓风》是一项关于在2017年玛利亚飓风事件中的遇难人数的调查。2017年9月20日，四级飓风“玛利亚”登陆拥有340万人口的美属岛屿波多黎各，对当地建筑与景观造成严重破坏，并产生电力与通讯中断、山体滑坡、洪水等影响。在12月波多黎各官方公布的受害者名单中，受害者人数为64人；然而，根据CPI、哈佛大学等机构的调查与研究，飓风“玛利亚”的死亡人数可能遭到严重低估，这一发现引发了国内对波多黎各官方及其统计机构的批评。2018年8月28日，波多黎各政府公布并承认了委托研究机构的调查结论，将在飓风“玛利亚”中直接或间接死亡的人数修正为2975人。

- 作品链接：https://hurricanemariasdead.com/

- 讲稿：[汇报：玛利亚飓风.docx](https://github.com/data-journalism/data-journalism.github.io/files/7133145/default.docx)

- PPT：[《谁死于玛利亚飓风》PPT.pptx](https://github.com/data-journalism/data-journalism.github.io/files/7133143/PPT.pptx)

- [Dataset of 1008 deaths](https://github.com/data-journalism/data-journalism.github.io/files/7142187/thing.txt)

**Comment by chengjun** [comment]
【数据分析&意义】汇报：玛利亚飓风
		1、简介
		2、数据来源 
			（1）真实性
			（2）视角：细腻➕人文关怀
			（3）和政府数据的对比
		3、数据分析
			（1）方格累计图
			（2）底层数据库
			（3）真实的句子
			（4）死亡率对比
			（5）可视化风格
			（6）颜色搭配
			（7）叙事特点
		4、意义

1、简介
（1）奖项背景
2019年全球数据新闻奖（Data Journalism Award）于当地时间6月14日在希腊揭晓。共有来自26个国家的607件作品参与评选，其中13件作品获奖。其中，由来自美联社、Quartz、和CPI（调查性新闻中心）的30多名记者共同完成的《谁死于玛利亚飓风》（Hurricane Maria’s Dead）获得年度调查报告奖。’

（2）事件背景
《谁死于玛利亚飓风》是一项关于在2017年玛利亚飓风事件中的遇难人数的调查。2017年9月20日，四级飓风“玛利亚”登陆拥有340万人口的美属岛屿波多黎各，对当地建筑与景观造成严重破坏，并产生电力与通讯中断、山体滑坡、洪水等影响。在12月波多黎各官方公布的受害者名单中，受害者人数为64人；然而，根据CPI、哈佛大学等机构的调查与研究，飓风“玛利亚”的死亡人数可能遭到严重低估，这一发现引发了国内对波多黎各官方及其统计机构的批评。2018年8月28日，波多黎各政府公布并承认了委托研究机构的调查结论，将在飓风“玛利亚”中直接或间接死亡的人数修正为2975人。

（3）新闻内容
Quartz、美联社与波多黎各CPI共同完成了一项关于飓风“玛利亚”遇难者身份与死因的调查，最终确定了504名受害者的身份与死因。同时，调查表明，灾后死亡人数的增加与政府的不作为有关。

报道首先以时间轴的形式展现了2017年9月至2018年6月实际死亡人数的变化，并与官方报告人数进行了对比，同时回顾了这10个月内官方的行动与部分遇难者家属的叙述。

随后，报道对2017年与2018年波多黎各的死亡率数据进行了对比，结果显示，在飓风“玛利亚”后，波多黎各的死亡人数激增，死于事故与心脏病等疾病的比例大幅增加。根据CPI的调查，灾后遇难者分别死于缺电、自然灾害、通讯中断、无法获得医疗服务、缺乏事物与水等原因。对不同年龄组灾后死亡率的分析显示，多数年龄组的死亡率在灾后上升，30-44岁为重灾区，事故与心脏病发作是这一年龄段死亡人数增加的原因之一。

除了新闻报道，该机构也开放了其调查的数据库，记载了遇难者的死亡日期、年龄、死亡地点、官方公布的死亡原因以及家属的叙述。读者可以自行在数据库中进行检索与筛选。

2、数据来源

（1）公众性
该调查的主要数据来源有两项，一项是波多黎各官方的死亡率数据，另一项是由受害者家属通过在线问卷提供的信息。这些信息在后续访谈中得到核实与完善。通过这种“众包新闻”的模式，新闻的参与主体从机构扩大到新闻当事人，受害者家属以数据提供者的身份参与了新闻生产，进而真正地参与公共事务的管理。

该调查也是多人协作生产新闻的典范，来自3家机构的30多名记者参与了该项目，采访了约300名死者家属。互联网降低了数据共享的成本，将媒体从单纯的内容生产平台转化为数据的集成加工平台，实现了新闻生产力的整合，提高了机构与机构之前、机构与公众之间协作生产新闻的效率。 

（2）真实性
当我们讨论一则数据新闻，我们需要思考两个问题：
为什么人们需要这个故事？
数据在这个故事中意味着什么？
在“玛利亚”飓风事件中，灾后对遇难者人数及死因的统计有助于当地改进公共管理措施、基础设施建设与应急预案，为未来的灾难性风暴再一次来临做好准备。然而在报道发布时，波多黎各政府并未将遇难者的死亡原因向医院或公众发布，也未及时更新遇难者数据。当单一的官方信源呈现出不可靠、不透明的特征时，从其他渠道挖掘与分析相关的数据，就成为揭示真实情况的重要手段。
CPI的数据来自遇难者家属的问卷反馈与访谈，这些数据与波多黎各政府为回应诉讼而公布的死亡证明进行了匹配。针对死亡证明，CPI也根据疾控中心的灾难相关死亡证明标准进行了审查。从材料来源上，该调查的数据弥补了政府公开数据的空缺，具有一定的真实性；从程序上，该调查保证了一定的严谨性，提高了数据的可信度。

（3）人文关怀
不同于高度概括的官方统计数据，也不同于海量的交易记录，《谁死于玛利亚飓风》数据的总体规模较小，但是每一条数据都包括遇难者的基本信息与家属叙述，这使得该数据库内的每一条数据都具有较高的信息密度与情感浓度，数据高度贴近个人与当时的情境，从而使整个项目彰显出“关注每一位遇难者”的细腻视角与尊重生命的人文关怀。

3、数据分析
（1）方格累计图
在逝者的统计上，本作品采用Waffle Chart（方格累计图）的方式来呈现。随着浮标的下划，故事的时间不断向前推进，每个个体的死亡记录以小方格的样式不断增加累积。
从时间线的角度来看，作者没有采用常规的时间线处理方法，例如横向的时间变动图，而是采取随着手势下滑，时间线不断推进的交互方法，细节之处体现良好的用户交互性。
从可视化的方法来看，作者没有采用折线图、饼图这类常规图案，而是采用一个个小方格的累计，这体现了作者对每一个逝者的尊重，也让读者能更加直接地面对一个个鲜活生命的消逝。这里如果使用折线图或柱状图，读者只能看到一个悬在空中的数字；而采用小方格的方法，让读者能够透过图形，看到一个个生命的离去，像是一具一具冰冷的尸体，不断压在读者的心头。
 

从叙事的角度来看，作者把故事冗杂在小方格之中，随着用户的手势下滑，自然地浮现出对应的小方格背后的真实的故事。使得故事和个体紧密的结合，这种叙事方式灵活且自然，不会让人有说教感。
 


（2）底层数据库
除了可视化展现，作者同样给出了底层的数据库，让读者可以直接看到每一个生命背后的具体的故事和细节。
从检索的角度，作者设计了【关键词检索】和【标签检索】，其中，标签按照死亡时间、死亡原因、死亡地点、居住点、年龄分类，具有良好的交互性。
 

从每条数据的设计来看，作者为每一位逝者制作了【卡片】，记录了逝者的死亡时间、年龄、死亡地点、官方死亡原因和死亡细节，以及数据来源。
其中，最有力量的部分来自作者对逝者死亡细节的补充。官方的死亡原因基本较为空泛，使用较多的专业术语；作者通过与逝者家人的访谈，补充了逝者的死亡细节，包括心理状态、动作细节、环境描写等内容，使得读者能够身临其境，增强代入感和共鸣感。
 

（3）真实的句子
基于底层数据库，作者收集了不同死亡原因下，逝者的死亡细节，主要采用第一人称的视角，描述逝者当时的心理状态。
相比于冰冷的数据，一句一句真实的又夹杂着痛苦的句子，使得读者更容易感同身受，理解每一位逝者背后的痛苦和挣扎。从而进一步增强故事的共鸣感和代入感。
从交互性上看，每个句子均来自底层数据库，用户可以通过点击，跳转到句子对应的主人，进一步了解这位逝者的更多细节，操作设计便利简洁。
 


（4）死亡率对比
除了灾难直接造成的伤亡，作者还切换了两个不同的视角，展现飓风对当地居民造成的后续影响。
第一，作者对比了飓风导致的死亡率和其余事故导致的死亡率。发现飓风事故导致很多常见疾病的发病率提升，例如心脏病、自杀、糖尿病等。
 

第二，作者统计了不同年龄段人群灾后的死亡率，发现绝大多数年龄段的灾民在灾难后的死亡率提升，尤其是30~34岁的人群，死亡率提升近40%。
以上两个图展现了作者对灾后人民的人性关怀，灾难使得当地不同年龄段人群的死亡率和部分疾病的死亡率提升，这或许和灾难对人的心理创伤、生理创伤或亲友离世造成的伤害相关。
相比于绝大多数媒体只把视角落在灾难现场的破坏性，本作品不仅展现了现场的受害者，也把饱含关怀的视角投向了灾后的人群。对于我们这些旁观者来说，灾难停止或许便是过去，但对于他们而言，灾难代表着亲友离世和无尽的恐惧，或许是一生难以割裂的痛苦伤疤。

（5）可视化风格
总体而言，整个作品的可视化没有采用很多花哨的图表，而是用最质朴的条形图和方格图，以贴合事实为目标，展现每一个个体的故事和灾后的持续性影响。
同时，作者在可视化过程中，也没有刻意回避“偏见”，灾难后，虽然成年和老年人群的死亡率上升，但婴儿的死亡率实实在在地降低了；同时，尽管意外事故、心脏病等疾病的死亡率上升，但癌症、脑血管疾病、杀人等问题的发生率确实降低了，这些作者都没有刻意略去，而是客观地呈现，供大家批判。
在数据的呈现上，不难感受到作者的真诚和对事实的尊重。

（6）颜色搭配
整个作品的颜色安排纯粹间接，以灰色和红色作为两个主要色调。
其中，灰色作为主要背景颜色，随着读者浏览与故事推进而自然地从浅灰色过渡到深灰色，增强视觉的层次性；同时，灰色作为主基调，象征着对逝者的哀悼，使观众更容易带入沉痛压抑的情绪；在逝者的经历记录部分，作者选择用灰色和白色的混合搭配进行记录，也代表着逝者的无声的缄默和灾难面前的苍白无力。
【插入一段视频】
除此之外，遇难者的经历采用红色，在灰色背景里显得格外醒目。让人想起《辛德勒的名单》中的红衣小女孩，代表着生命和控诉。
 

在本作品中，红色代表着每一个在灾难中逝去的生命，也代表血淋淋残酷的事实。
通过政府统计人数与实际人数对比，突出对政府掩埋真相的控诉。
 

除此外，红色或许寄托着对生命的尊重，希望读者能够感受生命的宝贵和不易。

（7）叙事特点
最后，从叙事特点上看，本作品非常重视叙事的连续性，所有的可视化以服务于故事的叙述为主，而没有喧宾夺主。
开头，作者交代故事的发生背景，并给出故事的核心观点“如果没有对悲剧的反思，那么悲剧将会再次发生”。
 

随后，全文的叙述铺陈了大量的文字，确保故事的连续性，同时衔接可视化的展现，不会显得生硬和割裂。
 

作为数据新闻，本作品没有刻意采用一些花哨的可视化方式；恰恰相反，本作品只在三个地方进行了可视化，其余部分更多的是对故事细节的披露和描述，可视化更多的发挥点缀和辅助说明的作用，真正实现了完整的故事线梳理。故事的代入感非常的强，给读者带来深刻的影响。
对比其它的一些作品，虽然围绕一个主题展开，但发散式的可视化使得图片之间的连接性很差，观众难以带入。最终只能感觉是一个很花哨的可视化技巧堆积，但没有形成完整且深入的故事，最终难以给读者留下深刻的印象。

4、意义
这部作品以飓风灾难中的逝者为主要的调查对象，关注每一个个体在灾难中真实的遭遇和故事，弥补了政府数据的单一和片面，为读者展现了窥探现实的另一个角度。
从可视化方法和细节上，这部作品可以说是质朴简洁但掷地有声，不仅展现每一个个体的细节心理，也关注灾难后的人群状态，可以感受到作者浓厚且真诚的人文关怀。
从启发意义角度来看，无论中外，在应对重大灾难面前，反思和善后工作都仍然不够完善。政府出于社会、政治、经济、外交等原因，一般不会披露关于死者的细节，也不会详细补充死者的死亡原因，从而尽可能地降低政府的担责。这部作品为我们提供了一个具有教育意义和启发意义的视角，我们可以把这种方式应用在更多灾难性事件上，吸引公众更加关注遇难者，了解遇难者背后的真实故事和死亡细节，剖析灾难应对过程中的不足举措，从而让反向倒逼全社会的应对重大灾难能力的提升。
正如作者在作品一开始提到的，“如果没有对悲剧的反思，那么同样的悲剧将会再次发生”。
谢谢大家！
